src/app/static/src/js/components/workflowWizard/workflowWizard.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+    <div id="workflow-wizard" class="container">
+        <step v-for="step in steps"
+              :key="getCurrentIndex(step.name)"
+              :active="isActive(step.name)"
+              :hasVisibility="handleVisibility(step.name)"
+              :valid="validStep"
+              @back="back(step.name)"
+              @next="next(step.name)"
+              @cancel="confirmCancel">
+            <component :is="step.component"
+                       @valid="handleStepValidity"
+                       :workflow-metadata="runWorkflowMetadata">
+            </component>
+        </step>
+        <cancel-dialog @cancel="cancel" @abortCancel="abortCancel" :show-modal="showModal"/>
+    </div>
+</template>
+
+<script lang="ts">
+    import Vue from "vue"
+    import step from "../workflowWizard/step.vue"
+    import {RunWorkflowMetadata, Steps} from "../../utils/types"
+    import runWorkflowReport from "../runWorkflow/runWorkflowReport.vue"
+    import runWorkflowRun from "../runWorkflow/runWorkflowRun.vue"
+    import cancelDialog from "../runWorkflow/cancelDialog.vue"
+
+    interface Data {
+        activeStep: number
+        validStep: boolean
+        showModal: boolean
+    }
+
+    interface Methods {
+        isActive: (name: string) => boolean
+        next: (name: string) => void
+        back: (name: string) => void
+        cancel: () => void
+        confirmCancel: () => void
+        abortCancel: () => void
+        handleVisibility: (name: string) => {}
+        getCurrentIndex: (name: string) => number
+        handleStepValidity: (valid: Event) => void
+    }
+
+    interface Props {
+        runWorkflowMetadata: RunWorkflowMetadata | null
+        backButtonVisible: boolean
```

Comment:
I don't understand this - I can't imagine a wizard without any back buttons. Shouldn't it just be hidden automatically for first step?